bigtable/client/chrono_literals.h

```diff
@@ -0,0 +1,37 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef BIGTABLE_CLIENT_CHRONO_LITERALS_H_
+#define BIGTABLE_CLIENT_CHRONO_LITERALS_H_
+
+#include <chrono>
+
+// TODO(coryan) - these are generally useful, consider submitting to abseil.io
+namespace bigtable {
+namespace chrono_literals {
+constexpr std::chrono::milliseconds operator "" _ms(unsigned long long ms) {
```

Comment:
FWIW user-defined literals aren't allowed in google3 without a special build flag.

Reply:
Looks like this is https://google.github.io/styleguide/cppguide.html#Operator_Overloading:

> #### Decision:
> [...] Do not overload `operator""`, i.e. do not introduce user-defined literals.

Sigh.

Reply:
Okay, but I do not see this as a showstopper.  We are already using exceptions, user-defined literals (since we cannot have C++1[47]) is trivial in comparison.

Reply:
I don't think `google3/third_party` is subject to Google C++ style guide. Also, there are already instances of `operator""` in `google3/third_party` so I think we're fine.